IdentityCore/src/util/ios/MSIDKeychainUtil.m

```diff
@@ -98,7 +98,11 @@ - (NSString *)appDefaultAccessGroup
             || readStatus == errSecInteractionNotAllowed)
         {
             NSMutableDictionary* addQuery = [query mutableCopy];
+#if TARGET_OS_UIKITFORMAC
+            [addQuery setObject:(id)kSecAttrAccessibleAfterFirstUnlock forKey:(id)kSecAttrAccessible];
```

Comment:
What's our story for coexistence and cache sharing between UIKit macOS and macOS apps? It seems we're planning to use iOS keychain code for TARGET_OS_UIKITFORMAC, but would that be right for SSO? Adding @rohitnarula7176

Reply:
As far as I understand, we cannot get keychain SSO between UiKitForMac & MacOS apps because they are using different keychains. We can follow up with Apple on this. Safari SSO should still work in case of system browser (we need to verify this).

Reply:
As we discussed offline, we have a separate work item to investigate SSO between UiKitForMac & MacOS.